app_dart/test/request_handlers/refresh_chromebot_status_test.dart

```diff
@@ -74,6 +74,31 @@ void main() {
       expect(task.status, Task.statusNew);
     });
 
+    test('do not update task status when commitSha/ref is unknown', () async {
+      final Commit commit = Commit(key: config.db.emptyKey.append(Commit, id: 'abc'), sha: 'abc');
```

Comment:
Here and below please add a "," at the end of the function calls to format the blocks in a more readable way.

Reply:
Done.